lib/Doctrine/ODM/MongoDB/Mapping/Driver/XmlDriver.php

```diff
@@ -335,15 +338,36 @@ public function loadMetadataForClass($className, \Doctrine\Persistence\Mapping\C
             }
         }
 
-        if (! isset($xmlRoot->{'also-load-methods'})) {
-            return;
+        if (isset($xmlRoot->{'also-load-methods'})) {
+            foreach ($xmlRoot->{'also-load-methods'}->{'also-load-method'} as $alsoLoadMethod) {
+                $metadata->registerAlsoLoadMethod((string) $alsoLoadMethod['method'], (string) $alsoLoadMethod['field']);
+            }
         }
 
-        foreach ($xmlRoot->{'also-load-methods'}->{'also-load-method'} as $alsoLoadMethod) {
-            $metadata->registerAlsoLoadMethod((string) $alsoLoadMethod['method'], (string) $alsoLoadMethod['field']);
+        if (isset($xmlRoot->{'time-series'})) {
+            $attributes = $xmlRoot->{'time-series'}->attributes();
+
+            $metaField          = isset($attributes['meta-field'])
+                ? (string) $attributes['meta-field']
+                : null;
+            $granularity        = isset($attributes['granularity'])
+                ? Granularity::from((string) $attributes['granularity'])
+                : null;
+            $expireAfterSeconds = isset($attributes['expire-after-seconds'])
+                ? (int) $attributes['expire-after-seconds']
+                : null;
```

Comment:
Is it split on multiple lines bacause of CS rule? That seems too much.

```suggestion
            $metaField          = isset($attributes['meta-field']) ? (string) $attributes['meta-field'] : null;
            $granularity        = isset($attributes['granularity']) ? Granularity::from((string) $attributes['granularity']) : null;
            $expireAfterSeconds = isset($attributes['expire-after-seconds']) ? (int) $attributes['expire-after-seconds'] : null;
```

Reply:
This was my attempt at making it more readable. I've changed it to single-line.